src/services/animals.js

```diff
@@ -19,19 +21,19 @@ class Animals {
     return animal;
   }
 
-
   async create(data, params) {
-    const Model = require('../models/animal');
-    let Animaldata = {
-      id: "id",
-      stage: "meme",
-      name: "name",
-      birthday: "birthday",
-      description: "description",
-      ingress_date: "ingress_date"
+    const AnimalData = {
+      stage: data.stage,
+      name: data.name,
+      dateOfBirth: data.dateOfBirth,
+      description: data.description,
+      entryDate: data.entryDate
     };
-    const animal = new Model(Animaldata);
-    this.animals.push(animal);
+
+    const animal = new AnimalModel(AnimalData);
```

Comment:
If you get `Cannot POST /animals` error (but then see it persists anyway) I think the error happens in this line. I should investigate mongoose more to understand what's wrong.